crypto/cpu-aarch64-linux.c

```diff
@@ -57,6 +95,13 @@ void OPENSSL_cpuid_setup(void) {
   if (hwcap & kSHA256) {
     OPENSSL_armcap_P |= ARMV8_SHA256;
   }
+
+  const char *env;
+  env = getenv("OPENSSL_armcap_P");
+  if (env == NULL) {
+    return;
+  }
+  handle_cpu_env(&OPENSSL_armcap_P, env);
```

Comment:
Maybe instead we should do:

```
if (env != NULL) {
    handle_cpu_env(&OPENSSL_armcap_P, env);
}
```

For additional clarity

Reply:
done